src/components/RightSidebar/Participants/ParticipantsTab.vue

```diff
@@ -121,13 +123,25 @@ export default {
 		},
 	},
 
-	setup() {
+	setup(props) {
 		const { sortParticipants } = useSortParticipants()
 		const isInCall = useIsInCall()
+		const {
+			participantsInitialised,
+			cancelableGetParticipants,
+			pendingChanges,
+			initialiseGetParticipants,
+			stopGetParticipants,
+		} = useGetParticipants(props.isActive)
 
 		return {
 			sortParticipants,
 			isInCall,
+			participantsInitialised,
+			cancelableGetParticipants,
+			initialiseGetParticipants,
+			pendingChanges,
```

Comment:
Same here for watcher on `isActive`. You probably need to make it reactive inside the composable, but thus you don't have to export `pendingChanges` and `debounceUpdateParticipants` (which you forgot btw)